Sources/PrimerSDK/Classes/User Interface/OAuth/OAuthViewController.swift

```diff
@@ -23,42 +23,39 @@ internal class OAuthViewController: PrimerViewController {
     deinit {
         log(logLevel: .verbose, message: "🧨 destroyed: \(self.self)")
     }
+    
+    func renderSpinner() {
+        view.addSubview(indicator)
+        let theme: PrimerThemeProtocol = DependencyContainer.resolve()
+        indicator.color = theme.colorTheme.disabled1
+        indicator.translatesAutoresizingMaskIntoConstraints = false
+        indicator.centerYAnchor.constraint(equalTo: view.centerYAnchor).isActive = true
```

Comment:
Effectively, you're adding those constraints each time you call `renderSpinner()`. You won't have constraints conflicts though. Reasons are:

- It's the same constraints
- `renderSpinner()` is only called once from the `viewDidLoad`

Reply:
👍